linked_list.rb

```diff
@@ -0,0 +1,83 @@
+Node.Struct.new(:word, :meaning, :next)
+class LinkedLists
+  attr_accessor :head, :tail
+
+  def initialize(first_node = nil)
+    @head = first_node
+    @tail = first_node
+  end
+
+  def add_first_node(word, meaning)
+    @head = Node.new(word, meaning, nil)
+    @tail = @head
+  end
+
+  def find_node(index)
+    counter = 0
+    current_node = @head
+    while counter < index
```

Comment:
you can eliminate `counter` variable by using `index.times` instead of `while` loop